zoo/example.txt

```diff
@@ -0,0 +1,24 @@
+Basic usage:
+
+Set up of a simple zoo with two cages for testing.
+
+from zoo.zoo import Zoo, Cage, BaseSpecies, Lion, Hyena, Wildebeest, Gazelle
+zoo = Zoo('Test Zoo')
+zoo.__repr__  # Displays the name of the zoo alongside the current number of cages and animals
```

Comment:
Just typing `zoo` should do it.  But if you want to get the repr within Python (rare), you should do `repr(zoo)`.